queries.sql

```diff
@@ -133,4 +133,11 @@ JOIN species specie ON animal.species_id = specie.id
 WHERE vet.name = 'Maisy Smith'
 GROUP BY specie.name
 ORDER BY visits DESC
-LIMIT 1;
\ No newline at end of file
+LIMIT 1;
+
+explain analyze SELECT COUNT(*) FROM visits where animal_id = 4;
+
+explain analyze SELECT vet_id FROM visits where vet_id = 2;
+
+explain analyze SELECT * FROM owners where email = 'owner_18327@mail.com';
+
```

Comment:
- [ ] As requested by the previous reviewer, kindly run your queries a number of times until explain analyze gives you a result with a time of lower than 1 second and update your pull request description with the screenshots.